src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx

```diff
@@ -0,0 +1,74 @@
+---
+subject: "Docs"
+releaseDate: '2024-06-28'
+version: 'version: June 21-27, 2024'
+---
+
+### New docs
+
+* Mobile monitoring's [application not responding](/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding) helps you track and analyze when and why your Android application's blocked for more than five seconds.
+* New browser monitoring [log](/docs/browser/new-relic-browser/browser-apis/log) and [wrapLogger](/docs/browser/new-relic-browser/browser-apis/wraplogger) API docs define the syntax, requirements, parameters, and examples on how to use these new endpoints.
+
+### Major changes
+
+* We've removed our top-level `Other capabilities` category and migrated everything that was there into more appropriate categories. This should make it somewhat easier for you to find what you're looking for.
+* Moved the OpenTelemetry collector examples on our docs site to our  `newrelic-opentelemetry-examples`](https://github.com/newrelic/newrelic-opentelemetry-examples) GitHub repository to make maintenance easier and improve their open-source visibility.
+
+### Minor changes
+
+* Added IAST validator service URL endpoints to [FedRAMP compliant endpoints](/docs/security/security-privacy/compliance/fedramp-compliant-endpoints/#agents) and [networks](/docs/new-relic-solutions/get-started/networks/#new-relic-endpoints).
+* Updated [IAST troubleshooting](/docs/iast/troubleshooting/#see-my-application) with specific IP ranges to whitelist if you're not seeing your application.
+* Updated [Ruby agent configuration](/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration/#instrumentation-aws_sqs) with `NEW_RELIC_INSTRUMENTATION_AWS_SQS` auto-instrumentation configuration information.
+* Removed `ApplicationExitInfo` reporting configuration information from [configure mobile monitoring settings](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings), because you can do this in the UI.
+* [Workflow enrichments](/docs/alerts/get-notified/incident-workflows/#enrichments) support multiple variables so you can customize your data more specifically ot your needs.
+* Added the [MobileApplicationExit](/attribute-dictionary/?dataSource=Mobile&event=MobileApplicationExit) attribute to our data dictionary. Our data dictionary isn't only a docs resource, but it also feeds attribute definitions directly to the UI, for example when you mouseover chart elements.
```

Comment:
```suggestion
* Added the [`MobileApplicationExit`](/attribute-dictionary/?dataSource=Mobile&event=MobileApplicationExit) attribute to our data dictionary. Our data dictionary isn't only a docs resource, but it also feeds attribute definitions directly to the UI in the NRQL query builder.
```

Also, love the shoutout here for data dictionary flow. But, I don't think it works for chart elements—‚only in query builder AFAIK.

Reply:
Good to know!